addons/crm_claim/crm_claim.py

```diff
@@ -38,13 +43,63 @@ class crm_claim(osv.osv):
     _name = "crm.claim"
     _description = "Claim"
     _order = "priority,date desc"
-    _inherit = ['mail.thread']
+    _inherit = ['mail.thread', 'ir.needaction_mixin']
 
     def _get_default_stage_id(self, cr, uid, context=None):
-        """ Gives default stage_id """
+        if context is None:
```

Comment:
here you are not doing any operation in context. so I think, you can remove this stuff